db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[8.0].define(version: 2025_04_22_085303) do
+ActiveRecord::Schema[8.0].define(version: 2025_04_25_134654) do
```

Comment:
The version here (ending in `134654`) does not match the version from the migration added here (ending `134308`) ? Maybe other local work/branches/changes interacted?